app/services/bulk_import_service.rb

```diff
@@ -182,4 +184,32 @@ def import_lists!
       [row.id]
     end
   end
+
+  def import_account_notes!
+    rows_by_acct = extract_rows_by_acct
+
+    if @import.overwrite?
+      @account.account_notes.reorder(nil).find_each do |account_note|
+        row = rows_by_acct.delete(account_note.target_account.acct)
+
+        if row.nil?
+          account_note.destroy!
+        else
+          row.destroy
+          @import.processed_items += 1
+          @import.imported_items += 1
+          @note = AccountNote.find_or_initialize_by(account: @account, target_account: row.data['acct'])
+          @note.comment = row.data['comment']
+          @note.save! if @note.changed?
```

Comment:
I think this should be in the `bulk_import_row_service`, not here, but I don't understand what I actually want to do here.